components/src/core/DropdownToolbar/DropdownToolbar.tsx

```diff
@@ -0,0 +1,190 @@
+            marginRight: theme.spacing(4),
+            cursor: 'pointer',
+        },
+        textContent: {
+            overflow: 'hidden',
+            whiteSpace: 'nowrap',
+        },
+        title: {
+            lineHeight: 1,
+            textOverflow: 'ellipsis',
+            overflow: 'auto',
+        },
+        subtitleContent: {
+            display: 'flex',
+            flexDirection: 'row',
+            maxWidth: 'fit-content',
+            cursor: 'pointer',
+        },
+        subtitle: {
+            overflow: 'hidden',
+            textOverflow: 'ellipsis',
+        },
+        arrowDropdown: {
+            marginLeft: 4,
+        },
+        rotateArrowDropdown: {
+            transform: 'rotate(180deg)',
+        },
+    })
+);
+
+export type DropdownToolbarClasses = {
+    root?: string;
+    navigation?: string;
+    textContent?: string;
+    title?: string;
+    subtitleContent?: string;
+    subtitle?: string;
+    arrowDropdown?: string;
+    menuItem?: string;
+};
+
+export const DropdownToolbar: React.FC<DropdownToolbarProps> = (props) => {
+    const { title, subtitle, menuItems, navigationIcon, customMenu, classes = {}, ...toolbarProps } = props;
+    const [anchorEl, setAnchorEl] = useState(null);
+    const anchor = useRef(null);
+    const theme = useTheme();
+    const defaultClasses = useStyles(theme);
+    const getNavigationIcon = useCallback(() => {
+        if (navigationIcon) {
+            return <div className={clsx(defaultClasses.navigation, classes.navigation)}>{navigationIcon}</div>;
+        }
+    }, [navigationIcon]);
+
+    const getMenu = useCallback(() => {
+        if (menuItems && Boolean(anchorEl)) {
+            return (
+                <Menu
+                    elevation={0}
+                    getContentAnchorEl={null}
+                    anchorOrigin={{ vertical: 'bottom', horizontal: 'left' }}
+                    transformOrigin={{ vertical: 'top', horizontal: 'left' }}
+                    anchorEl={anchorEl}
+                    open={Boolean(anchorEl)}
+                    onClose={(): void => setAnchorEl(null)}
+                    MenuListProps={{ style: { padding: 0 } }}
+                    PaperProps={{
+                        style: {
+                            maxHeight: MENU_ITEM_HEIGHT * 4.5,
+                        },
+                    }}
+                >
+                    {!customMenu &&
+                        menuItems.map((item: ToolbarMenuItem, index: number) => (
+                            <MenuItem
+                                key={`Toolbar-Option-${index}`}
+                                onClick={(): void => {
+                                    setAnchorEl(null);
+                                    item.onClick();
+                                }}
+                                className={clsx(classes.menuItem)}
+                            >
+                                {item.label}
+                            </MenuItem>
```

Comment:
See comment above about using menuGroups instead of menuItems for greater flexibility (and matching the UserMenu)....this will become a DrawerNavGroup with DrawerNavItems in it...thus using InfoListItem